xc7/boards.cmake

```diff
@@ -46,6 +46,15 @@ add_xc7_board(
   PART xc7a35tcsg324-1
 )
 
+add_xc7_board(
+  BOARD arty-full
+  DEVICE xc7a50t
+  PACKAGE test
+  PROG_TOOL ${OPENOCD_TARGET}
+  PROG_CMD "${OPENOCD} -f ${PRJXRAY_DIR}/utils/openocd/board-digilent-basys3.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
```

Comment:
`board-digilent-basys3.cfg` is used for the Arty?

Reply:
It seems that all our boards are using the config file named `board-digilent-basys3.cfg` ([see here](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/xc7/boards.cmake)) however it is so generic that it is suited for all of them.